src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,159 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    private static class Node<T> {
+        T value;
+        Node<T> next;
+        Node<T> prev;
```

Comment:
Don't use getters and setters in class Node. Since `Node<T>` is an inner class, you can access its private fields directly in the `MyLinkedList` class.